docs/UserGuide.md

```diff
@@ -33,95 +39,143 @@
+> Key in </priority> to show all tasks with specified priority
+<br><img src="images/ListHP.jpeg" width="600"><br>
+
+#### Deleting a task in CMDo : `delete`
+Deletes the specified task from the to do list.
+Format: `delete INDEX`
+
+> Deletes the person at the specified `INDEX`. 
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `list`
+  `delete 2`
+  Deletes the 2nd task in the to do list.
+* `list email` 
+  `delete 1`
+  Deletes the 1st task in the results of the `find` command.
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+#### Update task : `update`
+Update existing tasks in the CMDo list, as you would entering a new task.<br>
+Format: `edit INDEX details`  
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+> Examples: 
+> * `edit 2 eat candy tomorrow`
+>  Edit the 2nd task in the to do list to eat candy with date due as tomorrow 
+>  
+> * `edit 1 Bear`
+> Task 2 changes to Bear
 
-> The search is case sensitive, the order of the keywords does not matter, only the name is searched, 
-and persons matching at least one keyword will be returned (i.e. `OR` search).
 
-Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+####  Mark a task as done : `done <index>`
+Done a task so tick it off the list
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+Format: `done <index>`  
+<br><img src="images/Done.jpeg" width="600"><br>
+<br><img src="images/Done1.jpeg" width="600"><br>
 
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
 
-Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
-
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
-
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+#### Undo the last action : `undo`
+Undo the earlier action
+Format: `undo`  
+
+<br><img src="images/Undo.jpeg" width="600"><br>
+<br><img src="images/Undo1.jpeg" width="600"><br>
```

Comment:
***👍*** Do take note on the "highlight" on your UI and remember to implement it ~ user feedback is very impt ! I'm glad your team have taken consideration to it at such an early stage!